doc/content/verification_and_validation/val-2c.md

```diff
@@ -114,34 +113,117 @@ The case and model parameters used in both approaches in TMAP8 are listed in [va
 | $K_S^w$                    | $\boldsymbol{3.5 \times 10^{-4}} \times 6.0 \times 10^{24}$ | $\boldsymbol{3.0 \times 10^{-4}} \times 6.0 \times 10^{24}$ | 1/m$^3$/Pa | Adapted from [!cite](longhurst1992verification) |
 | $t_{injection}$            | N/A                                                          | 3                                                            | hr         |                                                 |
 
-## Results and discussion
+The calibration study was performed using [MOOSE's stochastic tools module](https://mooseframework.inl.gov/modules/stochastic_tools/index.html), and in particular the [Parallel Subset Simulation](https://mooseframework.inl.gov/source/samplers/ParallelSubsetSimulation.html) (PSS) approach.
```

Comment:
```suggestion
The calibration study was performed using [MOOSE's stochastic tools module](modules/stochastic_tools/index.md), and in particular the [Parallel Subset Simulation](samplers/ParallelSubsetSimulation.md) (PSS) approach.
```

Was there a particular reason why HTTPS was used here instead of the markdown files?